lua/modules/linear_bolzmann_solvers/do_transient/lbts_lua_utils.h

```diff
@@ -1,18 +0,0 @@
-// SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/>
-// SPDX-License-Identifier: MIT
-
-#pragma once
-
-#if 0
-#include "framework/chi_lua.h"
-
-namespace lbs::lbts_lua_utils
-{
-int LBSCreateTransientSolver(lua_State* L);
-int LBTSSetProperty(lua_State* L);
-int LBTSGetProperty(lua_State* L);
-int LBTSAdvanceTimeData(lua_State* L);
-
-void RegisterLuaEntities(lua_State* L);
-} // namespace lbs::lbts_lua_utils
-#endif
```

Comment:
just pinging @zhardy-lanl because this deals with transient. maybe it is not needed anymore. just want to make sure

Reply:
Yeah, I think I got a bit too aggressive. I should put that back...

Reply:
I double-checked this and these functions do not exist anywhere in the code base, i.e. these are just declarations  + registration of non-existing functions.

Do you still want to keep this?

Reply:
Sorry for the delay... These should be okay to get rid of IMO. When we get a transient solver up and running it will need to be hooked up to the `InputParameter` system and Lua namespaces anyways.